examples/hf_serialization.py

```diff
@@ -60,23 +110,40 @@ def serialize_model(
             is purely optional, and it allows for multiple models to be
             serialized to the same directory. A good example are Stable
             Diffusion models. Default is "model".
+        force: Force upload serialized tensors to `output_prefix`
+            even if they already exist
     """
 
-    os.makedirs(model_directory, exist_ok=True)
-    dir_prefix = f"{model_directory}/{model_prefix}"
+    if model_prefix is None:
+        model_prefix = "model"
 
+    dir_prefix = f"{model_directory}/{model_prefix}"
+    config_file_exists, weights_file_exists = (
+        check_file_exists(f"{dir_prefix}-config.json"),
+        check_file_exists(f"{dir_prefix}.tensors"),
+    )
     if config is None:
         config = model
     if config is not None:
```

Comment:
This existed before the change, but is very fishy. `config` will never be `None` by the time it gets to the second `if` statement. Does this code actually work if `config` starts out as `None` (e.g. for `diffusers`)?

Reply:
Not sure as this PR is only intended to titularly perform HF serialization.

Reply:
Yes, however, this function is shared between both parts of this general `tensorizer` example for both the HF and DF codepaths. If it worked before due to the contents of the second `if` block, but doesn't work now, that would be a regression and need to be fixed to be compatible with both styles before it was merged, since we can't break it for one to improve the other.